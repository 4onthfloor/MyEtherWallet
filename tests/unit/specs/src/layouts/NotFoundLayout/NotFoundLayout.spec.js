import { mount } from '@vue/test-utils';
import NotFoundLayout from '@/layouts/NotFoundLayout/NotFoundLayout.vue';
import { Tooling } from '@@/helpers';

describe('NotFoundLayout.vue', () => {
  let localVue, i18n, wrapper, store;

  beforeAll(() => {
    const baseSetup = Tooling.createLocalVueInstance();
    localVue = baseSetup.localVue;
    i18n = baseSetup.i18n;
    store = baseSetup.store;
  });

  beforeEach(() => {
    wrapper = mount(NotFoundLayout, {
      localVue,
      i18n,
      store,
      attachToDocument: true,
      mocks: {
        _t: () => {},
        $t: () => {}
      }
    });
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
