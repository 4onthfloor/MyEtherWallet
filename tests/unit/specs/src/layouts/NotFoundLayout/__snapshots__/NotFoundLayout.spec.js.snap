// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundLayout.vue renders correctly 1`] = `
<div class="not-found">
  <div class="wrap">
    <div class="page-container">
      <div class="content-block">
        <div class="block-title">404</div>
        <div class="block-text"></div>
        <div class="ailien-spaceship"><img src="~@/assets/images/icons/alien-spaceship.svg" alt=""></div>
      </div>
    </div>
  </div>
</div>
`;
